xml/docu_styleguide.language.xml

```diff
@@ -444,16 +453,17 @@
   <para>
    Use quotations to quote from sources, such as books.
    In other cases, avoid using quotation marks.
-   Do not use irony and do not use quotation marks to mark irony.
-   In the case of computer input, computer output, and user
-   interface elements, use more appropriate markup. See also
+   For computer input, computer output, and user
+   interface elements, use different markup. See also
    <xref linkend="sec-command-line"/> and
    <xref linkend="sec-ui-item-markup"/>.
+   Also, do not use quotation marks to indicate irony. Avoid irony in
+   technical writing.
   </para>
```

Comment:
Ultimately, in this case, I think it makes sense to go for a list, to avoid the weird cross-references in the middle of the text. Otherwise, I didn't change much except add a reference to sec-tone:

```suggestion
   In all other cases, do not use quotation marks:
  </para>
  <itemizedlist>
   <listitem>
    <para>
     For computer input, computer output, and user
     interface elements, use different markup.
     See also <xref linkend="sec-command-line"/> and
     <xref linkend="sec-ui-item-markup"/>.
    </para>
   </listitem>
   <listitem>
    <para>
     Do not use quotation marks to indicate irony.
     Avoid irony in technical writing.
     See also <xref linkend="sec-tone"/>.
    </para>
   </listitem>
  </itemizedlist>
```

Reply:
Thanks, I have created a separate PR: see https://github.com/SUSE/doc-styleguide/pull/166